test/micro/org/openjdk/bench/java/lang/foreign/AllocTest.java

```diff
@@ -23,27 +23,12 @@
 
 package org.openjdk.bench.java.lang.foreign;
 
-import org.openjdk.jmh.annotations.Benchmark;
-import org.openjdk.jmh.annotations.BenchmarkMode;
-import org.openjdk.jmh.annotations.Fork;
-import org.openjdk.jmh.annotations.Measurement;
-import org.openjdk.jmh.annotations.Mode;
-import org.openjdk.jmh.annotations.OutputTimeUnit;
-import org.openjdk.jmh.annotations.Param;
-import org.openjdk.jmh.annotations.Setup;
-import org.openjdk.jmh.annotations.State;
-import org.openjdk.jmh.annotations.TearDown;
-import org.openjdk.jmh.annotations.Warmup;
-
-import java.lang.foreign.Arena;
-import java.lang.foreign.FunctionDescriptor;
-import java.lang.foreign.Linker;
-import java.lang.foreign.MemorySegment;
+import java.lang.foreign.*;
```

Comment:
I believe it would be better to leave the imports alone -- as all the other benchmarks also use the "more explicit" style.

Reply:
I have expanded the import section, please reapprove this PR, thanks a lot.